### DISCUSSION TITLE
Add more keyboard shortcuts

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#8407] [Feature Requests]
From @benhaynes:

META-ENTER — Submit comments (and modals?)
META-A — Select all items in current Layout
META-S — Save and stay (overrides browser save)
META-W — Use browser close, but confirm before leaving if unsaved changes
I think we already have this
ARROWS — Navigate items on the layout, left/right can be used for nested lists
SPACE — Select the current item in a Layout
ENTER — Open the current item in a Layout
/ — Open search
? — Open advanced filter
ESC — Close current modal or drawer (one level, prompt for save as needed)
SHIFT-N — Create an item (not sure about this key combo)
SHIFT-ESC — Logout (not sure about this key combo)
META-' — Start a comment (not sure about this key combo, liked that it's like a quote, and near the shortcut to submit the comment)
\ — Open Field Context Menu for currently focused field
OPTION-1-9 — Go to Module in order
[ and ] — Previous next pages on Layout
. — Toggle info sidebar open/closed
, — Toggle page full-screen (not sure about key)
? — Show tooltips in popover (not built yet)

## Others

* All typical rich-tech formatting in WYSIWYG editors (including markdown)
* Confirm all form tab indexing is handled properly
* All dropdown menus items should show relevant keyboard shortcuts
* Something to navigate the navigation. We'll need a soft-select state (spoiler, I hate the browser "outline" thing) to show what you are on before activating

**Comment by joselcvarela** [comment]
Just found this https://bugs.chromium.org/p/chromium/issues/detail?id=33056
Basically there are some shortcuts we cannot use. In my case I was trying to override `CMD + N` but seems it is not possible.

**Reply by benhaynes** [reply]
Ahh, good to know!

**Comment by jiri-matejka** [comment]
Shortcut for top-right "Search items" would be briliant. I edit many content items and I am missing it a lot.

**Reply by benhaynes** [reply]
I see a lot of `/` used for a _Search_ shortcut... that seems like an easy one.

**Reply by jiri-matejka** [reply]
Shortcut `/` brings *Quick find* however which does not filter items. I mean normal top-right search box that filters out non-matching items.

**Reply by benhaynes** [reply]
Does it do quick find now?? I'm not seeing that, but I know it was discussed. We can discuss which maps to what... but I agree it's a good shortcut to add.

**Reply by staminna** [reply]
Can I customize ENTER Key top submit and update a form? I haven't seen where to set this options.

**Reply by azrikahar** [reply]
> Can I customize ENTER Key top submit and update a form? I haven't seen where to set this options.

There is no option to configure/customize keyboard shortcuts as of now. However, there is keyboard shortcuts for **Save and Stay** & **Save and Create New** (here `Ctrl` is depicted for Windows, whereas it will be `⌘` on Mac):

![image](https://user-images.githubusercontent.com/42867097/194983350-1107f4d9-8f45-43c2-b113-633b608da86b.png)

**Comment by staminna** [comment]
Thanks!

A terça, 11/10/2022, 03:31, Azri Kahar ***@***.***> escreveu:

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by arladmin** [comment]
I would also like to request a Keyboard shortcut for "Save as Copy".
